### PR TITLE
feat(vscode): create Visual Studio Code extension project

### PR DESCRIPTION
**Pull request by alestiago**
## Status

**READY**

## Description

Creates empty Visual Studio Code extension project using the official template.

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: extensions/vscode/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[![Dart Frog Logo][logo_white]][dart_frog_link_dark]
+[![Dart Frog Logo][logo_black]][dart_frog_link_light]
+
+[VSCode](https://code.visualstudio.com/) support for [Dart Frog](https://dartfrog.vgv.dev/) and provides tools for effectively creating Routes.
```
Comment (scarletteliza) [inline extensions/vscode/README.md:4]:
```suggestion
This is the [Dart Frog](https://dartfrog.vgv.dev/) [VSCode](https://code.visualstudio.com/) extension, which provides tools for effectively creating routes.
```

File: extensions/vscode/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Developed with 💙 by [Very Good Ventures][very_good_ventures_link] 🦄
+
+## Installation
+
+Dart Frog can be installed from the [VSCode Marketplace]() or by [searching within VSCode](https://code.visualstudio.com/docs/editor/extension-gallery#_search-for-an-extension).
```
Comment (scarletteliza) [inline extensions/vscode/README.md:10]:
```suggestion
Dart Frog can be installed from the [VS Code Marketplace]() or by [searching within VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_search-for-an-extension).
```
Are we missing a link here to the marketplace?

Comment (alestiago) [reply extensions/vscode/README.md:10]: Will add one once publicly released.

File: extensions/vscode/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+[![Dart Frog Logo][logo_white]][dart_frog_link_dark]
+[![Dart Frog Logo][logo_black]][dart_frog_link_light]
+
+This is the [Dart Frog](https://dartfrog.vgv.dev/) [VS Code](https://code.visualstudio.com/) extension, which provides tools for effectively creating routes.
+
+Developed with 💙 by [Very Good Ventures][very_good_ventures_link] 🦄
+
+## Installation
+
+Dart Frog can be installed from the [VS Code Marketplace]() or by [searching within VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_search-for-an-extension).
```
Comment (scarletteliza) [inline extensions/vscode/README.md:10]:
```suggestion
Dart Frog can be installed from the [VS Code Marketplace](https://marketplace.visualstudio.com/vscode) or by [searching within VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_search-for-an-extension).
```

**Review by scarletteliza — APPROVED** [review]

**Comment by brajas-quinnox** [comment]
Will there be a similar feature for android studio/intellij?

**Comment by alestiago** [comment]
We do currently plan to support a similar feature for Android Studio/IntelliJ 😄

**Pull request merged**